lib/client/api.go

```diff
@@ -463,18 +461,16 @@ type Config struct {
 
 	// DTAuthnRunCeremony allows tests to override the default device
 	// authentication function.
-	// Defaults to "dtauthn.NewCeremony().Run()".
 	DTAuthnRunCeremony DTAuthnRunCeremonyFunc
 
 	// dtAttemptLoginIgnorePing and dtAutoEnrollIgnorePing allow Device Trust
 	// tests to ignore Ping responses.
 	// Useful to force flows that only typically happen on Teleport Enterprise.
 	dtAttemptLoginIgnorePing, dtAutoEnrollIgnorePing bool
 
-	// dtAutoEnroll allows tests to override the default device auto-enroll
+	// DTAutoEnroll allows tests to override the default device auto-enroll
```

Comment:
This comment is no longer true, correct? We set this value in production code, not only in tests and we don't have the default value.
If yes, we should also update `DTAuthnRunCeremony`.